app/routes/jobs/job.py

```diff
@@ -37,7 +38,7 @@ async def get_job(*, job_id: UUID = Path(...)) -> UserJobResponse:
 
 
 async def _get_user_job(job_id: UUID) -> UserJob:
-    execution = _get_sfn_execution(job_id)
+    execution = await _get_sfn_execution(job_id)
```

Comment:
Wait, was this broken before?

Reply:
What do you mean? The PR was against another PR i had where I added this